plugins/database/src/db_plugin.cpp

```diff
@@ -15757,6 +15723,45 @@ auto db_update_ticket_write_byte_count_op(
     return SUCCESS();
 } // db_update_ticket_write_byte_count_op
 
+auto db_get_delay_rule_info_op(
+    irods::plugin_context&    _ctx,
+    const char*               _rule_id,
+    std::vector<std::string>* _info) -> irods::error
+{
+    if (const auto ret = _ctx.valid(); !ret.ok()) {
+        return PASS(ret);
+    }
+
+    if (!_rule_id || !_info) {
+        return ERROR(SYS_INVALID_INPUT_PARAM, "Invalid input: rule id or info container is null");
+    }
+
+    auto [db_instance, db_conn] = irods::experimental::catalog::new_database_connection();
+
+    nanodbc::statement stmt{db_conn};
+    nanodbc::prepare(stmt, "select rule_name, rei_file_path, user_name, exe_address, exe_time,"
+                           " exe_frequency, priority, last_exe_time, exe_status, estimated_exe_time,"
+                           " notification_addr, exe_context "
+                           "from R_RULE_EXEC where rule_exec_id = ?");
+
+    stmt.bind(0, _rule_id);
+
+    auto row = nanodbc::execute(stmt);
+
+    if (!row.next()) {
+        rodsLog(LOG_ERROR, "Could not find a delay rule with id [{}].");
+        return ERROR(CAT_NO_ROWS_FOUND, "Could not find a delay rule with id [{}].");
```

Comment:
Missing argument and `fmt::format` for the `{}`

Reply:
Done.